src/rpc/rawtransaction.cpp

```diff
@@ -345,24 +362,53 @@ UniValue getrawtransaction(const UniValue& params, bool fHelp)
             "         \"hex\"                     (string) output note commitment\n"
             "         ,...\n"
             "       ],\n"
-            "       \"onetimePubKey\" : \"hex\",  (string) the onetime public key used to encrypt the ciphertexts\n"
-            "       \"randomSeed\" : \"hex\",     (string) the random seed\n"
+            "       \"onetimePubKey\" : \"hex\",    (string) the onetime public key used to encrypt the ciphertexts\n"
+            "       \"randomSeed\" : \"hex\",       (string) the random seed\n"
```

Comment:
The extra whitespace in these cases is intentional, because it allows the generated output to line up correctly. It just looks weird in the source code, but please revert these changes.